docs/api/expect.md

```diff
@@ -701,6 +701,8 @@ You can provide an optional argument to test that a specific error is thrown:
 
 :::tip
 You must wrap the code in a function, otherwise the error will not be caught, and test will fail.
+
+> _This only applies to the synchronous usage !_
```

Comment:
1. The formatting doesn't follow docs style
2. This not makes it more confusing - I had to reread the issue to understand what you mean here.

This needs a proper explanation with an example